content/04.experiment-types.md

```diff
@@ -16,4 +16,146 @@ Proteomics experiments can be both qualitative and quantitative.
+
+2. Cellular Thermal Shift Assay (CETSA) [@doi:10.1016/j.cbpa.2019.11.004]
+CETSA is a powerful tool for studying protein-protein interactions in live cells. 
+Although CTSA has been widely used to study protein-protein interactions in a proteome-wide manner, little is known about the global changes of protein interaction states with the energy landscape. 
+The assay is capable of detecting shifts in the thermal equilibrium of cellular proteins in response to a variety of environmental perturbations, including the application of voltage, temperature, or magnetic fields. 
+This assay is able to detect both small and large changes in protein interaction states, making it an ideal tool for investigating the effects of environmental stressors on protein interactions.
+
+### Protein-protein interactions (PPIs)
+1. Affinity purification coupled to mass spectrometry (AP-MS) [@doi:10.1002/iub.1081; @doi:10.1038/nrg3356; @doi:10.1038/nmeth.2400]
+Affinity purification coupled to mass spectrometry (AP-MS)is an approach that involves the purification of a target protein or protein complex using a specific antibody or tag, followed by mass spectrometry analysis to identify the interacting proteins.
+In a typical AP-MS experiment, a protein or protein complex of interest is first tagged with a specific epitope or affinity tag, such as a FLAG or HA tag, that can be used to selectively capture the target protein using a corresponding antibody or resin. 
+The protein complex is then purified from the sample using a series of wash steps, and the interacting proteins are identified using mass spectrometry.
+The success of AP-MS experiments depends on a number of factors, including the quality of the antibody or tag used for purification, the specificity and efficiency of the resin used for capture, and the sensitivity and resolution of the mass spectrometer. 
+In addition, careful experimental design and data analysis are critical for accurate identification and interpretation of protein-protein interactions.
+AP-MS has been used to study a wide range of biological processes, including signal transduction pathways, protein complexes, and protein post-translational modifications. 
+Despite its widespread use, AP-MS has some limitations, including the potential for non-specific interactions, the difficulty in interpreting complex data sets, and the possibility of missing important interacting partners due to limitations in sensitivity or specificity. 
+However, with continued advances in technology and data analysis methods, AP-MS is likely to remain a valuable tool for the study of protein-protein interactions and for the development of new therapeutics.
+
+2. Avidity-based Extracellular Interaction Screen (APEX) [@doi:10.1186/1471-2105-9-529; @doi:10.1007/978-1-4939-9537-0_4; @doi:10.1155/2010/840518]
```

Comment:
```suggestion
2. APEX peroxidase [@doi:10.1038/nbt.2375; @DOI:10.1126/science.1230593]
```
The first and third citations were about an unrelated APEX method for protein quant